modules/javafx.controls/src/test/java/test/javafx/scene/control/cell/ParameterisedPrebuiltCellTest.java

```diff
@@ -69,10 +69,8 @@ public ParameterisedPrebuiltCellTest(Class<? extends Cell> cellClass) {
         count = 0;
 
         try {
-            cell = cellClass.newInstance();
-        } catch (InstantiationException e) {
-            e.printStackTrace();
-        } catch (IllegalAccessException e) {
+            cell = cellClass.getDeclaredConstructor().newInstance();
+        } catch (Exception e) {
```

Comment:
This will now catch and ignore runtime exceptions as well, where it previously would cause the test to fail. This could end up missing a real problem. The two additional checked exceptions, along with any runtime exceptions, should cause the test to fail. In fact, I don't know why the existing exceptions are ignored (unless there is a compelling reason to ignore them that I'm not aware of), so probably the best thing to do would be to eliminate the try / catch entirely, and instead add a `throws Exception` to the method declaration.

Reply:
Removed try/catch block and marked the method with `throws Exception`